pkg/scheduler/factory.go

```diff
@@ -181,7 +175,10 @@ func (c *Configurator) createFromProvider(providerName string) (*Scheduler, erro
 	defaultPlugins.Apply(c.plugins)
 	c.plugins = defaultPlugins
 
-	pluginConfig := []schedulerapi.PluginConfig{c.interPodAffinityPluginConfig()}
+	pluginConfig := []schedulerapi.PluginConfig{
```

Comment:
This was actually a temporary hack that we don't want to encourage.
It is going away with #88002

The correct solution would be to remove `BindTimeoutSeconds` from KubeSchedulerConfig. But we might be a little late for this in 1.18

What do other folks think?

Reply:
I agree to remove `BindTimeoutSeconds`. We can 1) move the volume binding logic into volumebinding plugin first (by using this temporary hack) and then 2) deprecate `BindTimeoutSeconds` in kube-scheduler component config. if 2) is late for 1.18, we can do it after 1.18.

Reply:
Ideally we should do it post 1.18 if we are not able to make it by then. However, I think we can backport it to older releases, it is a hack per se, and is fixed, it should be propagated to the supported releases

Reply:
I'd like to see this PR go with the elimination of ComponentConfig.BindTimeoutSeconds, @cofyc are you able to make it?

Reply:
sure